code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -660,6 +660,7 @@ GLOBAL_LIST_INIT(plastic_recipes, list(
 	new /datum/stack_recipe("large water bottle", /obj/item/reagent_containers/food/drinks/waterbottle/large/empty, 3), \
 	new /datum/stack_recipe("colo cups", /obj/item/reagent_containers/food/drinks/colocup, 1), \
 	new /datum/stack_recipe("wet floor sign", /obj/item/clothing/suit/caution, 2), \
+	new /datum/stack_recipe("warning cone", /obj/item/clothing/head/cone, 3), \
```

Comment:
```suggestion
	new /datum/stack_recipe("warning cone", /obj/item/clothing/head/cone, 2), \
```
If caution signs only cost 2 plastic sheets to craft, it's probably fair for warning cones to cost 2 plastic sheets to craft as well, especially since warning cones don't actually contain the plastic material and thus can't be recycled into plastic (as far as I can tell).